README.md

```diff
@@ -163,6 +164,7 @@ wsa://com.android.settings
 | Monument Valley | 2.7.17 | ✔️ |||
 | Monument Valley 2 | 2.0.3 | ✔️ |||
 | MT File Manager | 2.10.0 | ✔️ |||
+| Musically (TikTok) | 7.8.0 | ✔️ |||
```

Comment:
Can you please clarify the difference between Musically (TikTok) and these?
![image](https://user-images.githubusercontent.com/54220235/139596626-abdc6c90-5534-468e-8d0a-ac1e46e91587.png)

Reply:
musically is a really old version of tiktok, before it was tiktok.

Reply:
also musically allows signing in with a tiktok account but regular tiktok doesnt

Reply:
Alright.